department-of-veterans-affairs/va.gov-cms
# VACMS-1143: Update title help text on listing types.

## Description
See #1143 

## Testing done
Visual

## QA steps
As an admin, visit these pages (below) and visually verify that `Title field label` reads as `Page title`:
 - `/admin/structure/types/manage/story_listing`
 - `/admin/structure/types/manage/publication_listing`
 - `/admin/structure/types/manage/press_releases_listing`
 - `/admin/structure/types/manage/locations_listing`
 - `/admin/structure/types/manage/leadership_listing`
 - `/admin/structure/types/manage/health_services_listing`
 - `/admin/structure/types/manage/event_listing`
